src/Tizen.Telephony/Tizen.Telephony/Modem.cs

```diff
@@ -130,8 +136,14 @@ public PowerStatus CurrentPowerStatus
         /// <summary>
         /// Gets the MEID (Mobile Equipment Identifier) of a mobile phone (for CDMA).
         /// </summary>
+        /// <remarks>
+        /// For avoding the unexpected behavior of old version applications that have http://tizen.org/privilege/telephony privilege. There is an exceptional handling in case of permission denied.
```

Comment:
Spelling, should be `avoiding`. It is correct, but I suggest using `To avoid` instead.  It should be one sentence.

Reply:
Done